colin-api/src/colin_api/resources/filing.py

```diff
@@ -296,3 +296,35 @@ def post(identifier, **kwargs):
         except Exception as err:  # pylint: disable=broad-except
             current_app.logger.error(f'Error updating AR status for {identifier}: {str(err)}')
             return jsonify({'message': 'Error updating AR status.'}), HTTPStatus.INTERNAL_SERVER_ERROR
+ 
+ 
+@cors_preflight('POST')
+@API.route('/<string:legal_type>/<string:identifier>/filings/reminder')
```

Comment:
Please change route to @API.route('/<string:identifier>/filings')

Reply:
@ozamani9gh is there any specific reason for this route? This is not correct to keep a generic route for a reminder delete. Ideally it should be a DELETE (instead of POST) and should have a specific route to delete ar reminder